### ISSUE TITLE
Comments iframe needs title attribute

### ISSUE THREAD

**Issue by danjjohnson** [Automattic/jetpack#9118]
Section 508 and WCAG 2.0 accessibility standards request/require title attributes to be included on iframe elements which describes the content of the iframe for screen readers, etc.

The iframe output by Jetpack comments does not include a title attribute.

Related to https://github.com/Automattic/jetpack/issues/7370

1014730-zen

**Comment by colleengreene** [comment]
Is there an ETA on when this will be included in a JetPack update? I have the latest version, and this issue is causing our websites to fail accessibility compliance. Thank you for your work on this!

**Comment by jeherve** [comment]
@colleengreene This was added to Jetpack in version 6.0. If you still see the issue on your site, could you contact us via [this form](https://jetpack.com/contact-support/?rel=support) so we can take a closer look at your site?

Thank you!
